docs/sources/query/logcli/logcli-tutorial.md

```diff
@@ -0,0 +1,409 @@
+logcli query '{service_name="Delivery World"} | package_status="critical"'
+```
+
+This will return all logs where the `service_name` is `Delivery World` and the `package_status` is `critical`. The output will look similar to the following:
+
+```console
+http://localhost:3100/loki/api/v1/query_range?direction=BACKWARD&end=1732617594381712000&limit=30&query=%7Bservice_name%3D%22Delivery+World%22%7D+%7C+package_status%3D%22critical%22&start=1732613994381712000
+Common labels: {package_status="critical", service_name="Delivery World"}
+2024-11-26T10:39:52Z {package_id="PKG79755", package_size="Small", state="Texas"}       {"timestamp": "2024-11-26T10:39:52.521602Z", "state": "Texas", "city": "Dallas", "package_id": "PKG79755", "package_type": "Clothing", "package_size": "Small", "package_status": "critical", "note": "In transit", "sender": {"name": "Sender38", "address": "906 Maple Ave, Dallas, Texas"}, "receiver": {"name": "Receiver41", "address": "455 Pine Rd, Dallas, Texas"}}
+2024-11-26T10:39:50Z {package_id="PKG34018", package_size="Large", state="Illinois"}    {"timestamp": "2024-11-26T10:39:50.510841Z", "state": "Illinois", "city": "Chicago", "package_id": "PKG34018", "package_type": "Clothing", "package_size": "Large", "package_status": "critical", "note": "Delayed due to weather", "sender": {"name": "Sender22", "address": "758 Elm St, Chicago, Illinois"}, "receiver": {"name": "Receiver10", "address": "441 Cedar Blvd, Naperville, Illinois"}}
+```
+
+Lets suppose we want to look back for the last 24 hours, we can use the `--since` flag to specify the time range:
+
+```bash
+logcli query --since 24h '{service_name="Delivery World"} | package_status="critical"' 
+```
+
+This will query all logs for the `package_status` `critical` in the last 24 hours. However it will not return all of the logs, but only the first 30 logs. We can use the `--limit` flag to specify the number of logs to return:
+
+```bash
+logcli query --since 24h --limit 100 '{service_name="Delivery World"} | package_status="critical"' 
+```
+
+### Metric Queries
+
+We can also use LogCLI to query logs based on metrics. For instance as part of the site report we want to count how many packages are being sent from California in the last 5 minutes. We can use the following query:
+
+```bash
+logcli query 'sum(count_over_time({state="California"}[5m]))'
+```
+
+Lets suppose we only want to know the number of packages of type `document` being sent from California in the last 5 minutes. We can use the following query:
+  
+```bash
+logcli query 'count_over_time({state="California"}| json | package_type= "Documents" [5m])'
+```
+
+### Instant Metric Queries
+
+Instant metric queries are a subset of metric queries that return the value of the metric at a specific point in time. This can be useful for quickly understanding an aggregate state of the stored logs. 
+
+For instance, we can use the following query to get the number of packages being sent from California at the current time:
```

Comment:
Your introductory text says this query is for "the current time" but it looks like it's for the past five minutes if you don't notice that it's an instant-query.  Maybe call out the change from `query` to `instant-query` in the text to highlight it?

Reply:
aaah thats a really good point will fix the wording there.